### DISCUSSION TITLE
v.generalize settings to emphasize 90-degree turns in building polygons

### DISCUSSION THREAD

**Discussion by DeyanVasilev** [OSGeo/grass#2792] [General]
Hello,

I am curious if somebody has good v.generalize settings to emphasize 90-degree turns in building polygons.

My polygons are being created out of a fine resolution raster with a step < 0.5 m. Resulting polygons have jigsaw-pattern sides and have to be simplified in order to get good-looking rectangles. Unfortunately all  v.generalize algorithms I tested tend to produce polygons with somehow uneven corners. That's why I'm looking for something that would emphasize 90-degree turns that are common to building polygons.

**Comment by wenzeslaus** [comment]
I would just try the algorithms with various settings on the specific data and see what works best.

There is no algorithm which would be specifically good for this case, however this behavior is something which would be a good and possible addition to the tool.
